lib/driver.js

```diff
@@ -930,6 +930,8 @@ class XCUITestDriver extends BaseDriver {
       return;
     }
 
+    await verifyApplicationPlatform(this.opts.app, this.isSimulator());
```

Comment:
So this is just for logging purposes at present?

Reply:
I don't know how reliable this architecture verification method is, because I was not able any proper documentation on it. Lets keep it as is for now. Later, we could easily change the method to throw an error ;)